code/modules/vending/clothingbooth_items.dm

```diff
@@ -46,6 +46,90 @@
 	category = "Accessories"
 	cost = PAY_TRADESMAN/3
 
+/datum/clothingbooth_item/hbowmagenta
```

Comment:
why are these all distinct subtypes instead of being commonly inherited?

Reply:
https://github.com/goonstation/goonstation/blob/987aee5b61d6f7327da2f4198cf8ee173f0f11e9/code/modules/vending/clothingbooth.dm#L8

You will need to change this line as well to support ZeWakas request to abstract and subtype the bows

Reply:
#5480